README.md

```diff
@@ -1,18 +1,22 @@
 # STAC Project Steering Committee
 
-This document describes how the STAC (SpatioTemporal Asset Catalog) Project Steering Committee makes decisions on STAC and STAC API project issues. STAC is concerned with both the [stac-spec](https://github.com/radiantearth/stac-spec) and [stac-api-spec](https://github.com/radiantearth/stac-api-spec) repositories.
+This document describes how the STAC (SpatioTemporal Asset Catalog) Project Steering Committee makes decisions on STAC and STAC API project issues. STAC is concerned with the following GitHub repositories and organizations:
+- [stac-spec](https://github.com/radiantearth/stac-spec)
+- [stac-api-spec](https://github.com/radiantearth/stac-api-spec)
+- [stac-extensions](https://github.com/stac-extensions)
+- [stac-api-extensions](https://github.com/stac-api-extensions)
 
 In brief the committee votes on proposals on <https://github.com/radiantearth/stac-psc/issues>. Proposals are available for review for at least five business days, and a single veto is sufficient to delay progress though ultimately a majority of members can pass a proposal.
 
 ## Members
 
 The list of members of the Project Steering Committee is:
 
+* Chris Holmes (chair, [@cholmes](https://github.com/cholmes))
```

Comment:
Just to reflect the current status, I think. We can change this after accepting this document once.